src/components/CallView/Grid/Grid.vue

```diff
@@ -1010,19 +1011,14 @@ export default {
 	height: 100%;
 	width: 100%;
 
-	grid-row-gap: 8px;
-	grid-column-gap: 8px;
+	grid-row-gap: var(--grid-gap);
```

Comment:
I remember there was a plan to add CSS variables for 8/12/16px. Maybe stick with calc value, to migrate easier later?

Reply:
Well, we can split into 3 local variables and rename it later to CSS variables.

Reply:
Whatever you find a good solution in that case